packages/material-ui-lab/src/TreeView/TreeView.js

```diff
@@ -66,6 +89,19 @@ const TreeView = React.forwardRef(function TreeView(props, ref) {
     selected: selectedProp,
     ...other
   } = props;
+  const theme = useTheme();
```

Comment:
```suggestion
  // use the `isRtl` from the props after the buildAPI script support it
  const theme = useTheme();
```

@oliviertassinari I remember there was issue with this, should we at least leave a comment for fixing it in the future?

Reply:
I put `useTheme` back and replaced `isRtl` by `theme.direction === 'rtl'`.